Dockerfile

```diff
@@ -28,7 +28,9 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
-ADD . /work
+COPY . /work
```

Comment:
What's the reason to use `COPY` instead of `ADD` here?

Reply:
hadolint had these warnings and that is the reason for the change https://github.com/hadolint/hadolint/wiki/DL3020